aiperf/timing/credit_issuing_strategy.py

```diff
@@ -81,15 +81,27 @@ def _setup_profiling_phase_config(self) -> None:
                 )
             )
         else:
-            self.debug(
-                f"Setting up count-based profiling phase: total_expected_requests={self.config.request_count}"
+            debug_message = (
```

Comment:
i think you should instead be setting a local `total_expected_requests` type thing where u compute it based on the config, and then only have one debug + append